docs/install.md

```diff
@@ -0,0 +1,132 @@
+#Installation and Settings
+The MongoDB module and sub-modules need some configuration to be useful. This
+guide assumes that :
+
+* a [MongoDB][download] 3.0 to 3.6 instance is already installed, configured, and
+  available to connect to from the Drupal instance.
+* the site will be running [Drupal][drupal] 8.[56].x, with [Drush][drush] 8.x.
```

Comment:
Should add a note about Drush 9/Console planned compatibility, linking to https://www.drupal.org/project/mongodb/issues/2986785